### ISSUE TITLE
Can't add new entry

### ISSUE THREAD

**Issue by Mxctr1** [buttercup/buttercup-mobile#344]
Once you add a new Vault, you cannot add an entry. as soon as you try, the screen becomes unresponsive. you can only swipe back and when you swipe back the app crash... 

**Steps to reproduce**

- Open Buttercup on IOS
- Enter in a Vault and try to enter a new Entry

**Comment by perry-mitchell** [comment]
Thanks, I can reproduce. I'll try to fix it this evening. Apologies for the inconvenience.

**Comment by perry-mitchell** [comment]
So this seems to be something more systemic, perhaps with UIKitten:

```
2023-12-04 21:41:30.819205+0200 Buttercup[46034:1895237] [Presentation] Attempt to present <RCTModalHostViewController: 0x7f9a657162d0> on <UIViewController: 0x7f9b24d08550> (from <RNSScreen: 0x7f9a65662950>) which is already presenting <RCTModalHostViewController: 0x7f9ac5132d90>.
```

The root one always shows (password for vault), but the nested ones, after this release, no longer do. Not sure what's causing it to be honest.

There are other issues on the UIKitten repo referencing a crash when switching tabs: https://github.com/akveo/react-native-ui-kitten/pull/1790 - but I don't think this is entirely related.

**Comment by perry-mitchell** [comment]
Tried downgrading UI kitten, and then upgrading React-native to the latest. No dice. It's some other internal library 😞

**Comment by perry-mitchell** [comment]
Released a fix in 2.6.1.

**Comment by Mxctr1** [comment]
Seems like the problem is when you try to save the application just crash
